### PR TITLE
Display the current ENF Version within our app information menu (EXPOSUREAPP-2794)

### PR DESCRIPTION
**Pull request by d4rken**
Display the current ENF version within our app information screen to support our community and make debugging efforts easier.

Closes https://github.com/corona-warn-app/cwa-wishlist/issues/190

Few more files changed, but if it's worth doing, it's worth doing right :wink:.

### CODE REVIEW

**Comment by vaubaehn** [comment]
Huh, didn't expect that many files to be changed...

But, thanks a lot :heart:

File: Corona-Warn-App/src/main/res/layout/fragment_information.xml [hunk 1]
```diff
@@ -117,11 +112,24 @@
                     android:layout_height="wrap_content"
                     android:layout_marginTop="@dimen/spacing_small"
                     android:focusable="true"
-                    android:text="@{FormatterInformationHelper.formatVersion()}"
+                    tools:text="v1.8.0-RC1"
```
Comment (BMItter) [inline Corona-Warn-App/src/main/res/layout/fragment_information.xml:115]: hardcoded? 
If this is intended to be this way, leave it there..
Just not sure about this..

**Review by BMItr — DISMISSED** [review]
nothing to complain. Tests succeeded.
Tested on Pixel 2 Device.
left one question.

**Comment by d4rken** [comment]
I forgot the click on the text, will add tomorrow.

File: Corona-Warn-App/src/main/res/layout/fragment_information.xml (hunk 1 above)
Comment (d4rken) [reply Corona-Warn-App/src/main/res/layout/fragment_information.xml:115]: `tools:text` is just displayed in the layout preview in Android Studio.

**Review by vaubaehn — COMMENTED** [review]
Simply awesome.
Anytime (after COVID) you get to HH, give me a sign and take your free 🍺 .

**Review by harambasicluka — APPROVED** [review]

**Review by BMItr — APPROVED** [review]

**Pull request merged**